01-node-tutorial/answers/11-fs-async.js

```diff
@@ -0,0 +1,24 @@
+// const { readFileSync, writeFileSync } = require('fs')
+const { writeFileSync }  = require('fs')
+
+writeFileSync("./temporary/fileB.txt" , `This is line 1\n This is line 2\n This is line 3\n`, (err, result) => {
```

Comment:
This technically does write three lines to a file, and in the correct place as well, but it's synchronous, and needs to be asynchronous. How could you use the callback, which happens after the first file is written with `writeFile()`, to write more files?